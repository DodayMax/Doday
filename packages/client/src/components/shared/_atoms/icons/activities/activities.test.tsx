import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Activities', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.Activities />);

    expect(icon).toMatchSnapshot();
  });
});
