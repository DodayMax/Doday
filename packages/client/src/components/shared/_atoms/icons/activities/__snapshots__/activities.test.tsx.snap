// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activities renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M12.098 2.178c-5.4 0-9.778 4.377-9.778 9.778 0 5.4 4.378 9.777 9.778 9.777s9.778-4.377 9.778-9.777c0-5.4-4.378-9.778-9.778-9.778zm8.422 9.333h-1.778a6.667 6.667 0 0 0-6.2-6.2V3.533a8.444 8.444 0 0 1 7.978 7.978zm-8.422 5.778a5.333 5.333 0 1 1 0-10.667 5.333 5.333 0 0 1 0 10.667zm-8.445-5.333A8.444 8.444 0 0 1 11.21 3.56v1.778a6.667 6.667 0 1 0 4.724 12.044l1.271 1.271a8.444 8.444 0 0 1-13.55-6.697zM18.2 17.782l-1.258-1.258a6.64 6.64 0 0 0 1.778-3.68h1.778a8.409 8.409 0 0 1-2.298 4.938z"
        fill="black"
        fillRule="evenodd"
      />,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M12.098 2.178c-5.4 0-9.778 4.377-9.778 9.778 0 5.4 4.378 9.777 9.778 9.777s9.778-4.377 9.778-9.777c0-5.4-4.378-9.778-9.778-9.778zm8.422 9.333h-1.778a6.667 6.667 0 0 0-6.2-6.2V3.533a8.444 8.444 0 0 1 7.978 7.978zm-8.422 5.778a5.333 5.333 0 1 1 0-10.667 5.333 5.333 0 0 1 0 10.667zm-8.445-5.333A8.444 8.444 0 0 1 11.21 3.56v1.778a6.667 6.667 0 1 0 4.724 12.044l1.271 1.271a8.444 8.444 0 0 1-13.55-6.697zM18.2 17.782l-1.258-1.258a6.64 6.64 0 0 0 1.778-3.68h1.778a8.409 8.409 0 0 1-2.298 4.938z",
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M12.098 2.178c-5.4 0-9.778 4.377-9.778 9.778 0 5.4 4.378 9.777 9.778 9.777s9.778-4.377 9.778-9.777c0-5.4-4.378-9.778-9.778-9.778zm8.422 9.333h-1.778a6.667 6.667 0 0 0-6.2-6.2V3.533a8.444 8.444 0 0 1 7.978 7.978zm-8.422 5.778a5.333 5.333 0 1 1 0-10.667 5.333 5.333 0 0 1 0 10.667zm-8.445-5.333A8.444 8.444 0 0 1 11.21 3.56v1.778a6.667 6.667 0 1 0 4.724 12.044l1.271 1.271a8.444 8.444 0 0 1-13.55-6.697zM18.2 17.782l-1.258-1.258a6.64 6.64 0 0 0 1.778-3.68h1.778a8.409 8.409 0 0 1-2.298 4.938z"
          fill="black"
          fillRule="evenodd"
        />,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M12.098 2.178c-5.4 0-9.778 4.377-9.778 9.778 0 5.4 4.378 9.777 9.778 9.777s9.778-4.377 9.778-9.777c0-5.4-4.378-9.778-9.778-9.778zm8.422 9.333h-1.778a6.667 6.667 0 0 0-6.2-6.2V3.533a8.444 8.444 0 0 1 7.978 7.978zm-8.422 5.778a5.333 5.333 0 1 1 0-10.667 5.333 5.333 0 0 1 0 10.667zm-8.445-5.333A8.444 8.444 0 0 1 11.21 3.56v1.778a6.667 6.667 0 1 0 4.724 12.044l1.271 1.271a8.444 8.444 0 0 1-13.55-6.697zM18.2 17.782l-1.258-1.258a6.64 6.64 0 0 0 1.778-3.68h1.778a8.409 8.409 0 0 1-2.298 4.938z",
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
